src/server/license/server/api_server.go

```diff
@@ -344,13 +346,48 @@ func (a *apiServer) UpdateCluster(ctx context.Context, req *lc.UpdateClusterRequ
 	a.LogReq(req)
 	defer func(start time.Time) { a.pachLogger.Log(req, resp, retErr, time.Since(start)) }(time.Now())
 
-	if err := a.validateClusterConfig(ctx, req.Address); err != nil {
-		return nil, err
+	// should we only validate the pachd Address accessible to the enterprise server?
```

Comment:
surfacing this comment/question:

should we only validate the pachd Address accessible to the enterprise server?

Reply:
I'm not sure I follow, is the question whether we should check whether the enterprise server can connect to the URI that the end users will get from syncing? I don't think so - the goal is to support a case where the users and the enterprise server have different network topologies, such that they reach the pachd via different addresses. In cases were the `enterprise -> pachd` and `client -> pachd` routes are different, I think it's likely we can't use the client URI to reach the pachd from the enterprise server.

Reply:
that makes total sense. I think I'm trying to get at whether there's some validation worth performing on the user_address. At least with the network accessible pachd address, we are confident in its validity. 

I guess the only thing we can really do is validate the protocol. Does that seem worth doing? I guess that's something that's implied by how we configure the server.

I think I answered my own question here lol - nothing to validate for that value. Feel free to chime in if I'm off.